lib/shared/permissions.js

```diff
@@ -101,7 +116,22 @@ exports.getEnvironmentCommandPrefix = (environment) => {
     }
 
     if (isWindows) {
-      return [ 'set', `${key}=${value}`, '&&' ];
+      const definition = `${key}=${value}`;
+      return [
+        'set',
+        _.attempt(() => {
+          if (/\s/.test(definition)) {
+            return quoteString(definition);
+          }
+
+          return definition;
+        }),
+        '&&'
+      ];
+    }
+
+    if (/\s/.test(value)) {
```

Comment:
I think that might be because it'll treat that everything after the command as an argument – should work if only called on the value though (instead of `quoteString()`)?
Also, why aren't the env vars passed through the `env` option to `sudoPrompt.execAsync()`? Does that not work properly on Windows?